vulkancapsviewer.cpp

```diff
@@ -1166,7 +1165,7 @@ void addVariantListItem(QStandardItem* parent, const QVariantMap::const_iterator
 {
     QList<QStandardItem*> item;
     item << new QStandardItem(iterator.key());
-    item << new QStandardItem(arrayToStr(iterator.value()));
+    new QStandardItem(arrayToStr(iterator.value()));

```

Comment:
What's, uh... what's goin' on here? Removing the stream output turns this into just an orphan allocation that's not assigned to anything, which I'm vaguely surprised isn't a compiler error. Did you mean this...?

```c++
QList<QStandardItem*> item;
item << new QStandardItem(iterator.key())
     << new QStandardItem(arrayToStr(iterator.value()));
```

Reply:
Yikes! Yeah, I'm surprised I "got away with this". Yes, both get added to item.